README.md

```diff
@@ -21,10 +21,6 @@ We are trying to minimize the amount of work to add or modify data going into `R
 - Modifying the destination `Redshift` tables, if necessary
 - Refreshing the latest `Redshift` data (see: the `granularity` flag) by efficiently loading s3 data using the `COPY` command
 
-## Deploying
-
-This is a standard mesos deploy, `ark start -e production s3-to-redshift`
```

Comment:
instead of deleting this, maybe we should say. 

This repo will auto deploy after you merge.  To do a manual deploy, use `ark start -e production s3-to-redshift`

(since the command is still accurate)

Reply:
👍